AmazonChimeSDK/AmazonChimeSDK/internal/utils/Converters.swift

```diff
@@ -28,6 +28,8 @@ import Foundation
                 return .meetingStartRequested
             case .meetingStartSucceeded:
                 return .meetingStartSucceeded
+            case .meetingReconnected:
```

Comment:
Do you mind pasting an example of the JSON blob of this event in the PR description?